language_parity_chart.md

```diff
@@ -6,15 +6,15 @@ These charts outline the various language library components and features implem
 |---	|---	|---	|---	|---	|---	|
 |AWS X-Ray Trace ID generation	|Done	|Done	|Done	|Done	|Done	|
 |AWS X-Ray Trace ID propogation and trace header web framework handling	|Done	|Done	|Done	|Done	|Done	|
-|Basic support for tracing call to AWS using language AWS SDK	|Done	|Done	|In progress	|Done	|In progress	|
+|Basic support for tracing call to AWS using language AWS SDK	|Done	|Done	|Done	|Done	|Done	|
 |Library metadata	|Done	|Done	|Done	|Done	|Done	|
 |DB/SQL support	|Done	|Done	|In review	|In review ([OTel Issue #159](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/159))	|Done	|
-|AWS X-Ray errors/exceptions format	|Done (via ADOT Collector)	|Done (via ADOT Collector)	|In OTel, need traslation done in collector	|Done (via ADOT Collector)	|Needs implementation in OTel ([OTel issue #1491](https://github.com/open-telemetry/opentelemetry-go/issues/1491))	|
+|AWS X-Ray errors/exceptions format	|Done (via ADOT Collector)	|Done (via ADOT Collector)	|Done (via ADOT Collector)	|Done (via ADOT Collector)	|Done (via ADOT Collector)	|
 |Resource Detectors - AWS Elastic Beanstalk	|Done	|Done	|EC2 (through ADOT Collector)	|EC2 (through ADOT Collector)	|Done	|
 |Resource Detectors - AWS EC2	|Done	|Done	|EC2 (through ADOT Collector)	|EC2 (through ADOT Collector)	|Done	|
-|Resource Detectors - AWS ECS	|Done	|Done	|Partial (through ADOT Collector) <br/> Container ID not recorded	|Partial (through ADOT Collector) <br/> Container ID not recorded	|Done	|
-|Resource Detectors - AWS EKS	|Done	|Done	|Partial (through ADOT Collector)|Partial (Through ADOT Collector)|Done	|
-|AWS X-Ray Trace ID injection into application logs	|Done	|	|	|	|	|
-|Metadata/Annotations	|Done (through ADOT Collector)	|Done (through ADOT Collector)	|Done (through ADOT Collector)	|Done (through ADOT Collector)	|Done (Through Collector)	|
-|AWS Lambda support	|	|	|	|Done (with autoinstumentation)	|	|
-|Auto-instumentation	|Done	|Needs implementation in OTel	|Needs implementation in OTel	|Done	|Needs implementation in OTel	|
+|Resource Detectors - AWS ECS	|Done	|Done	|Done	|Done	|Done	|
+|Resource Detectors - AWS EKS	|Done	|Done	|Done |Done |Done	|
+|AWS X-Ray Trace ID injection into application logs	|Done	| NA	| NA	| NA	| NA   |
+|Metadata/Annotations	|Done (through ADOT Collector)	|Done (through ADOT Collector)	|Done (through ADOT Collector)	|Done (through ADOT Collector)	|Done (Through ADOT Collector)	|
+|AWS Lambda support	|Done	|Done	| NA |Done (with auto-instrumentation)	| NA	|
+|Auto-instrumentation	|Done	|Done	|Needs implementation in OTel	|Done	|Needs implementation in OTel	|
```

Comment:
Can we make sur that wherever we have "Needs implementation in OTel" there is a link to the upstream issue to aid tracking?

Reply:
For NodeJS, I see that [aws-sdk](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-aws-sdk) and [aws-lambda](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-aws-lambda) and not any specific issues to it. 
Would like to double check if we are reflecting right status, if not I can modify the status for this DONE.

For .Net - We have listed a [TODO](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/5284642ae19884569a88bf44cca6e8e1ed276a9c/docs/internal/instrumentation-libraries.md#aws)  in the docs, but couldn't find any tracking [issue](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues?page=4&q=is%3Aissue+is%3Aopen) or dicsussions. 

CC: @willarmiros to help with any info that we might be missing here about tracking issues.